src/test/scala/chiselTests/ToTargetSpec.scala

```diff
@@ -82,6 +83,46 @@ class RelativeSiblingsModule extends RawModule {
   }
 }
 
+@instantiable
+class RelativeDefinitionGrandChildModule(relativeRoot: RawModule) extends RawModule {
+  @public val grandChildWire = Wire(Bool())
+}
+
+@instantiable
+class RelativeDefinitionChildModule(relativeRoot: RawModule) extends RawModule {
+  val inProgressChild = IO(Output(Property[Path]()))
+  val inProgressGrandChild = IO(Output(Property[Path]()))
+
+  @public val childWire = Wire(Bool())
+
+  @public val grandChild = Module(new RelativeDefinitionGrandChildModule(relativeRoot))
+
+  atModuleBodyEnd {
+    // Check relative targets created from an in-progress Definition.
+    val childReference = childWire.toRelativeTarget(Some(relativeRoot))
+    inProgressChild := Property(Path(childReference))
+
+    val grandChildReference = grandChild.grandChildWire.toRelativeTarget(Some(relativeRoot))
+    inProgressGrandChild := Property(Path(grandChildReference))
+  }
+}
+
+class RelativeDefinitionParentModule extends RawModule {
+  val completedChild = IO(Output(Property[Path]()))
+  val completedGrandChild = IO(Output(Property[Path]()))
+
+  val child = Instantiate(new RelativeDefinitionChildModule(this))
```

Comment:
When `Instantiate` two `RelativeDefinitionChildModule` do we need to elaborate twice?

Rather than passing `this` to `RelativeDefinitionChildModule`, what would happen if passing other modules to it?

I wonder, rather than passing it in parameter, how about passing the target from `Input(Property[Path]())`? This might keep the module parameter stable?